compiler/rustc_builtin_macros/src/autodiff.rs

```diff
@@ -65,9 +84,32 @@ mod llvm_enzyme {
             dcx.emit_err(errors::AutoDiffInvalidMode { span: meta_item[1].span(), mode });
             return AutoDiffAttrs::error();
         };
+
+        // Now we check, whether the user wants autodiff in batch/vector mode, or scalar mode.
+        // If he doesn't specify an integer (=width), we default to scalar mode, thus width=1.
+        let mut first_activity = 2;
+
+        let width = if meta_item.len() >= 3
+            && let Some(x) = width(&meta_item[2])
```

Comment:
```suggestion
        let width = if let [_, _, x, ..] = &meta_item[..]
            && let Some(x) = width(x)
```

Reply:
There's generally sth we could do more robustly (using `split_first` to keep shrinking it and taking elements out) around the meta item processing, but let's do that in follow-up PRs.